### PR TITLE
Create bat file to build Cecil for katana automation

### PR DESCRIPTION
**Pull request by mderoy**

### CODE REVIEW

**Comment by joshpeterson** [comment]
I hesitate to say this, but most of our other builds user perl as the build script. Is there a specific reason we should use a bat file for this one instead?

**Comment by TautvydasZilys** [comment]
@joshpeterson if that makes you feel better, I used a .bat for building roslyn too.

**Comment by joshpeterson** [comment]
@TautyvdasZilys that does make me feel a little better. If we are already doing this, then it is probably fine.

**Comment by mrvoorhe** [comment]
lgtm

**Review by joshpeterson — APPROVED** [review]

File: build.bat [hunk 1]
```diff
@@ -0,0 +1,42 @@
+set BASEDIR=%~dp0
+set BUILDDIR=%BASEDIR%builds
+set LIBDIR=%BUILDDIR%\lib
+set LIBDIR35=%LIBDIR%\net35
+set LIBDIR40=%LIBDIR%\net40
+set MSBUILD="C:\Program Files (x86)\MSBuild\14.0\Bin\MSBuild.exe"
```
Comment (joncham) [inline build.bat:6]: It would be nice to not hard code this, but not a big deal

**Comment by joncham** [comment]
LGTM

**Pull request merged**